build.ps1

```diff
@@ -34,11 +29,6 @@ elseif ($isReleaseBuild)
 {
     $buildCommand = { dotnet run --ci --generateSBOM }
```

Comment:
The 1ES pipeline should generate SBOM for you so I would consider removing any manual SBOM generation. Please see tips and tricks from the documentation

Reply:
Addressed it.